LICENSES/LicenseRef-Broadcom-RPi.txt

```diff
@@ -0,0 +1,29 @@
+Copyright (c) 2006, Broadcom Corporation.
```

Comment:
I've explained the situation of this file in an out of line comment. Let's discuss this there but this needs to be clarified.